src/features/attributeSelect.tsx

```diff
@@ -0,0 +1,343 @@
+      </div>
+    </Dialog>
+  );
+};
+const AttributeButton = ({
+  attributeName,
+  uid,
+}: {
+  attributeName: string;
+  uid: string;
+}) => {
+  return (
+    <Button
+      icon="chevron-down"
+      intent="primary"
+      minimal
+      onClick={(e) => {
+        const configUid = getPageUidByPageTitle(CONFIG);
+        const tree = getBasicTreeByParentUid(configUid);
+        const attributeUid = tree.find((t) => t.text === "attributes")?.uid;
+        const options = getSubTree({
+          key: attributeName,
+          parentUid: attributeUid,
+        }).children.map((t) => t.text);
+        renderOverlay({
+          Overlay: ChooseAttributeOverlay,
+          id: "attribute-select",
+          props: { tree, options, uid, attributeName },
+        });
+      }}
+    />
+  );
+};
+
+const renderAttributeButton = (
+  parent: HTMLSpanElement,
+  attributeName: string,
+  blockUid: string
+) => {
+  const containerSpan = document.createElement("span");
+  containerSpan.onmousedown = (e) => e.stopPropagation();
+  ReactDOM.render(
+    <AttributeButton attributeName={attributeName} uid={blockUid} />,
+    containerSpan
+  );
+  parent.appendChild(containerSpan);
+};
+
+const AttributeConfigPanel = () => {
+  const [value, setValue] = useState("");
+  const configUid = getPageUidByPageTitle(CONFIG);
+  const attributesUid = getSubTree({
+    key: "attributes",
+    parentUid: configUid,
+  }).uid;
+  const [definedAttributes, setDefinedAttributes] = useState<string[]>(
+    getDefinedAttributes()
+  );
+  const attributesInGraph = (
+    window.roamAlphaAPI.data.fast.q(
+      `[:find
+        (pull ?page [:node/title])
+      :where
+        [?b :attrs/lookup _]
+        [?b :entity/attrs ?a]
+        [(untuple ?a) [[?c ?d]]]
+        [(get ?d :value) ?s]
+        [(untuple ?s) [?e ?uid]]
+        [?page :block/uid ?uid]
+      ]`
+    ) as [PullBlock][]
+  )
+    .map((p) => p[0]?.[":node/title"] || "")
+    .filter((a) => !definedAttributes.includes(a));
+
+  return (
+    <div className={Classes.DIALOG_BODY}>
+      <div className="flex mb-8">
+        <div id="attribute-select-autocomplete">
+          <AutocompleteInput
+            value={""}
+            setValue={setValue}
+            options={attributesInGraph}
+            placeholder={"Choose Attribute To Add"}
+          />
+        </div>
+        <Button
+          intent="primary"
+          className="mx-2"
+          disabled={!value}
+          text={"Add Attribute"}
+          rightIcon={"plus"}
+          style={{ marginLeft: 16 }}
+          onClick={() => {
+            createBlock({
+              node: {
+                text: value,
+                children: [{ text: "" }],
+              },
+              order: "last",
+              parentUid: attributesUid,
+            }).then(() => {
+              setDefinedAttributes(definedAttributes.concat([value]));
+              // Automcomplete setting value to "" is not working
+              setValue("");
```

Comment:
not sure what's going on here
Tried different values for `value`, no `value`, removing the attribute from the DOM, it just keeps getting reset to whatever value is in there